content/build/release-notes.md

```diff
@@ -8,6 +8,101 @@ toc_max: 2
 This page contains information about the new features, improvements, and bug
 fixes in [Docker Buildx](https://github.com/docker/buildx).
 
+## 0.12.0
+
+{{< release-date date="2023-11-16" >}}
+
+The full release note for this release is available
+[on GitHub](https://github.com/docker/buildx/releases/tag/v0.12.0).
+
+### New
+
+- New `--annotation` flag for the `buildx build`, and an `annotations` key in the Bake file, that lets you add OCI Annotations to build results.
+  [#2020](https://github.com/docker/buildx/pull/2020),
+  [#2098](https://github.com/docker/buildx/pull/2098)
+- New experimental debugging features, including a new `debug` command and an interactive debugging console.
+  This feature currently requires setting `BUILDX_EXPERIMENTAL=1`.
+  [#2006](https://github.com/docker/buildx/pull/2006),
+  [#1896](https://github.com/docker/buildx/pull/1896),
+  [#1970](https://github.com/docker/buildx/pull/1970),
+  [#1914](https://github.com/docker/buildx/pull/1914),
+  [#2026](https://github.com/docker/buildx/pull/2026),
+  [#2086](https://github.com/docker/buildx/pull/2086)
+
+### Bug fixes and enhancements
```

Comment:
Not related but I still don't like mixing Bugs and enhancements :see_no_evil:

Reply:
same